mobile/library/swift/EngineBuilder.swift

```diff
@@ -68,6 +73,11 @@ open class EngineBuilder: NSObject {
   private var nodeRegion: String?
   private var nodeZone: String?
   private var nodeSubZone: String?
+#if canImport(EnvoyCxxSwiftInterop)
+  private(set) var useSwiftBootstrap = true
```

Comment:
I enabled it by default so that we would get test coverage for the new mode. But I suppose we already have some coverage with the assert in `bootstrapDebugDescription()`.

I'll enable it in the experimental Swift app too so we have at least some coverage there.